src/app/shared/components/icon/icon.component.html

```diff
@@ -0,0 +1,33 @@
+          fill="#005EA5"
+        ></path>
+      </g>
+      <defs>
+        <clipPath id="clip0_25_318"><rect width="100" height="56" fill="white"></rect></clipPath>
+      </defs>
+    </svg>
+  </ng-container>
+
+  <ng-container *ngSwitchCase="'trl'">
+    <svg width="100" height="56" viewBox="0 0 100 56" fill="none" xmlns="http://www.w3.org/2000/svg">
+      <path
+        d="M52.8345 40.8062C56.8027 40.8062 60.0907 43.6279 60.6576 47.3178H64.2857C64.8526 43.6279 68.1406 40.8062 72.1088 40.8062C76.4172 40.8062 80.0454 44.1705 80.0454 48.4031C80.0454 52.5271 76.5306 56 72.1088 56C68.1406 56 64.8526 53.1783 64.2857 49.4884H60.6576C60.0907 53.1783 56.8027 56 52.8345 56C48.5261 56 44.898 52.6357 44.898 48.4031C44.898 44.1705 48.5261 40.8062 52.8345 40.8062ZM72.1088 53.7209C75.1701 53.7209 77.7778 51.3333 77.7778 48.2946C77.7778 45.3643 75.2834 42.8682 72.1088 42.8682C69.0476 42.8682 66.4399 45.2558 66.4399 48.2946C66.5533 51.3333 69.0476 53.7209 72.1088 53.7209ZM52.8345 53.7209C55.8957 53.7209 58.5034 51.3333 58.5034 48.2946C58.5034 45.3643 56.0091 42.8682 52.8345 42.8682C49.7732 42.8682 47.1655 45.2558 47.1655 48.2946C47.1655 51.3333 49.7732 53.7209 52.8345 53.7209ZM80.6122 41.8915C80.6122 42.5426 81.0658 42.9767 81.746 42.9767H91.3832C92.0635 42.9767 92.517 42.5426 92.517 41.8915C92.517 41.2403 92.0635 40.8062 91.3832 40.8062H81.746C81.0658 40.8062 80.6122 41.2403 80.6122 41.8915ZM1.13379 0H98.8662C99.5465 0 100 0.434107 100 1.08527V35.9225C100 36.5736 99.5465 37.0078 98.8662 37.0078H15.1927V40.6977H44.2177C44.898 40.6977 45.3515 41.1318 45.3515 41.7829C45.3515 42.4341 44.898 42.8682 44.2177 42.8682H15.1928V53.0698C15.1928 53.7209 14.7392 54.155 14.059 54.155C13.3787 54.155 12.9252 53.7209 12.9252 53.0698V37.0078H1.13379C0.453526 37.0078 1.0134e-05 36.5736 1.0134e-05 35.9225V1.08527C1.30129e-05 0.542636 0.453526 0 1.13379 0ZM2.26758 34.8372H97.7324V2.17054H2.26758V34.8372Z"
+        fill="#005EA5"
+      ></path>
+    </svg>
+  </ng-container>
+
+  <ng-container *ngSwitchDefault>
```

Comment:
Make the default blank and have a case for PSV? Just in case we get an vehicle type we don't expect

Reply:
If the ticket or Bruce states the certainty of us having other vehicle types, then yes.

If not, I am against this as it would be coding for the future.